app/views/gift_requests/gift_request.slim

```diff
@@ -1,12 +1,16 @@
 - action = @gift_request.new_record? ? "create" : "edit"
 - url = @gift_request.new_record? ? gift_requests_new_wishlist_path : gift_request_path(@gift_request.id)
+
+div.wediko-back
```

Comment:
instead of copypasting this everywhere you can now just assign the 'back button' route to `@back_route` and it'll put the back button in automatically